### ISSUE TITLE
Remove link to old quickref

### ISSUE THREAD

**Issue by yatil** [w3c/wai-wcag-quickref#141]

**Comment by robustusability** [comment]
Wait until #138 is fixed. The current quickref fails in too many browsers, which is sad for a page about WCAG.

**Comment by yatil** [comment]
Can you please point out in which browsers the quickref fails?

**Comment by robustusability** [comment]
It fails in any browser that fails to load or execute the required JavaScript, e.g. my brand new Windows Phone and my older Sony (note that they have no options to turn on/off JavaScript). Read #138 and the link there. Changing the script logic would make it work regardless of JavaScript working or not. That is the point of "unobtrusiveness", which is implied in the robustness requirement in WCAG.
